BritneyS/comics-graphql
# Adding GraphQL

## What does this PR do :question:
This PR adds the GraphQL resolvers, including retrieving a character by the characterID.


## How to test it :microscope:
1. Pull down this branch and open the GraphQL Playground at `http://localhost:8000/graphql`.
2. Confirm that the following query returns all character objects as expected, according to the query criteria (see below)
```graphql
query{
  characters {
    id
    name
    placeOfBirth
    race
  }
}
```
Expected result:
```graphql
{
  "data": {
    "characters": [
      {
        "id": 1,
        "name": "A-Bomb",
        "placeOfBirth": "Scarsdale, Arizona",
        "race": "Human"
      },
      {
        "id": 2,
        "name": "Abe Sapien",
        "placeOfBirth": "-",
        "race": "Icthyo Sapien"
      },
      {
        "id": 3,
        "name": "Abin Sur",
        "placeOfBirth": "Ungara",
        "race": "Ungaran"
      },
      {
        "id": 4,
        "name": "Abomination",
        "placeOfBirth": "Zagreb, Yugoslavia",
        "race": "Human / Radiation"
      },
...etc
}
```
3. Confirm that the following query returns the record for the character "A-Bomb" as expected (see character object below):
```graphql
query{
  character(id: 1){
    id
    name
    placeOfBirth
    race
  }
}
```
Expected result:
```graphql
{
  "data": {
    "character": {
      "id": 1,
      "name": "A-Bomb",
      "placeOfBirth": "Scarsdale, Arizona",
      "race": "Human"
    }
  }
}
```


## Any background info you would like to include :white_check_mark:
N/A


## Screenshots (if applicable) :camera:
N/A
